rosdep/base.yaml

```diff
@@ -6694,6 +6694,9 @@ qhull-bin:
   gentoo: [media-libs/qhull]
   nixos: [qhull]
   ubuntu: [qhull-bin]
+qml-module-qt-labs-folderlistmodel:
+  debian: [qml-module-qt-labs-folderlistmodel]
+  ubuntu: [qml-module-qt-labs-folderlistmodel]
```

Comment:
I think that in fedora this is included in https://fedora.pkgs.org/34/fedora-x86_64/qt6-qtdeclarative-6.0.1-1.fc34.i686.rpm.html, though I'm not completely sure.

Reply:
I think the one with a matching Qt version would be `qt5-qtdeclarative`. It appears that only Debian provides fine grained Qt component dependencies.